src/index.js

```diff
@@ -87,6 +94,7 @@ function spawnBaddy(x, y, id) {
       size,
       speed,
       hp,
+      hitPlayer,
```

Comment:
To make the enemy have an attack rate, you should add a value to the `enemies` array for this enemy that defines how fast they can attack (see weapon [rate of fire](https://github.com/straker/js13k-2022/blob/main/src/tables/weapons.js#L8). Then in the baddy update function you can do what the [player does to apply the attack rate](https://github.com/straker/js13k-2022/blob/main/src/index.js#L55-L73) to the enemy attacks.